### DISCUSSION TITLE
Consulta PR - Actividad 3

### DISCUSSION THREAD

**Discussion by NicoPavoni** [frlp-utn-ingsoft/catedra#121] [Q&A] [answered]
Buenas! Tengo una consulta con respecto a la actividad 3.
Modifiqué el código agregando como una feature el manejo de stock para los productos.
Una vez hecho todo, hice el commit, pushee y despues hice el PR.
Cuando hice el PR se generó con todos los commits desde que arrancamos con el proyecto.  Esto es porque es el primer PR que hacemos ?
Está bien que se haga el PR en este caso ? O se debería generar otro mas limpio ?

Gracias.

**Comment by ivanduva** [comment, marked as answer]
Nico, hiciste un PR a nuestro repositorio, por eso aparecen todos los cambios. Tenés que hacerlo al repositorio del grupo.
Saludos
